src/Paramore.Brighter.ServiceActivator/MessagePump.cs

```diff
@@ -119,11 +123,26 @@ public void Run()
                 }
 
                 // Serviceable message
+                Activity span = null;
                 try
                 {
+                    message.UpdateCloudEventsFromHeaders();//ToDo: Discuss this as a temp measure
```

Comment:
I would suspect we want to establish headers that are populated as a result of translating a message i.e. to message headers from the transport, and then test for the presence of those headers i.e. MessageHeaders has some specific properties, and then we can figure out how to populate MessageHeaders from the transport, whether cloud events are present or not. binary vs structured etc

Reply:
Did we want to add these to the transport in 9.x as I'm happy to do that? 

I was working on the assumption of have them in the header Bag for v9.x then move them out with v10

The reason this is here is due to how the rmq transport reconstructed the message Headers and then was meant to be a shim until we went full implementation